samples/analyze.v1p3beta1.js

```diff
@@ -0,0 +1,362 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function detectPerson(path) {
+  //[START video_detect_person_beta]
+  // Imports the Google Cloud Video Intelligence library + Node's fs library
+  const Video = require('@google-cloud/video-intelligence').v1p3beta1;
+  const fs = require('fs');
+  // Creates a client
+  const video = new Video.VideoIntelligenceServiceClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const path = 'Local file to analyze, e.g. ./my-file.mp4';
+
+  // Reads a local video file and converts it to base64
+  const file = fs.readFileSync(path);
+  const inputContent = file.toString('base64');
+
+  const request = {
+    inputContent: inputContent,
+    features: ['PERSON_DETECTION'],
+    videoContext: {
+      personDetectionConfig: {
+        // Must set includeBoundingBoxes to true to get poses and attributes.
+        includeBoundingBoxes: true,
```

Comment:
You don't use the bounding boxes in the output, is it worth addding?

Reply:
The bounding boxes are necessary to get the person and face detection segments but don't need to be read. 

Reading bounding boxes is a pretty common task for the Video Intelligence API. On one hand, developers should be familiar with it already and this sample is really more about the attributes. On the other hand, it *is* a common task for using the API and maybe a good thing to include anyway.

I don't have a strong feeling one way or the other about bounding boxes, besides that adding them makes this sample longer. What is your preference?